.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 _agstate
 build
+yarn-error.log
+ui/public/lockdown.umd.js
```

Comment:
looks like that file is still in the repo: https://github.com/Agoric/dapp-amm/blob/c72887833df707be00c424eef0d91706af0d0ff3/ui/public/lockdown.umd.js

You'll have to explicitly delete it